typing/path.ml

```diff
@@ -107,3 +107,10 @@ let is_constructor_typath p =
   match constructor_typath p with
   | Regular _ -> false
   | _ -> true
+
+module T = struct
+  type nonrec t = t
+  let compare = compare
+end
+module Set = Set.Make(T)
+module Map = Map.Make(T)
```

Comment:
You could probably use `Identifiable.Make` for this instead.

Reply:
`Identifiable.Make` require (two) printers and an equal function. I would prefer to leave that for later.